fuzz/fuzz_targets/streams.rs

```diff
@@ -0,0 +1,22 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+use std::io;
+use bytes::{Bytes};
+
+extern crate proto;
+use proto::connection::{Streams};
+use proto::frame::{Stream};
+use proto::{Side, Dir, StreamId};
+
+
+fuzz_target!(|data: &[u8]| {
+    let mut client = Streams::new(Side::Client, 128, 128, 1024 * 1024, 1024 * 1024, 1024 * 1024);
```

Comment:
This is copy pasted straight from the streams test module and may not be the most useful target.